docs/contents/components/data-display/stat/accessibility.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Stat
+description: "`Stat` is used to display numbers or data within a box."
+package_name: "@yamada-ui/stat"
+order: 3
+with_description: true
+tab: Accessibility
+---
+
+## ARIA Roles and Attributes
+
+| Component  | Role and Attributes | Usage                                                                                                  |
+| ---------- | ------------------- | ------------------------------------------------------------------------------------------------------ |
+| `Stat`     | `icon`              | If set to `"increase"` or `"decrease"`, it is passed as `type` to `StatIcon`.                          |
+| `StatIcon` | `type`              | Can be set to `"increase"` or `"decrease"`.                                                            |
```

Comment:
There is no `type` role or attributes.This is unique to Yamada UI.Please remove this line.